locale/en/LC_MESSAGES/django.po

```diff
@@ -1241,59 +1239,59 @@ msgstr "You have not selected any other researchers."
 msgid ""
 "Indien je geen toestemming hebt van een andere ethische commissie, dien je "
 "het normale formulier in te vullen. Ga terug naar de startpagina, en "
-"selecteer \"Een nieuwe studie aanmelden (from scratch in een leeg "
-"formulier)\" of \"Een nieuwe studie aanmelden (vanuit een kopie van een oude "
-"studie)\"."
+"selecteer \"Een nieuwe aanvraag aanmelden (from scratch in een leeg "
+"formulier)\" of \"Een nieuwe aanvraag aanmelden (vanuit een kopie van een "
+"oude aanvraag)\"."
 msgstr ""
-"If you don't have formal approval from a different committee, you need to "
-"complete the regular registration form. Return to the start page, and select "
-"\"Register a new study (from scratch in an empty form)\" or \"Register a new "
-"study (from a copy of an old study)\"."
+"If you don't already have formal approval from a different ethics committee, "
+"you need to complete the regular registration form. Return to the start "
+"page, and select \"Submit a new proposal that is completely new in this "
+"portal\" or \"from a copy of an old proposal.\""
 
 #: proposals/forms.py:226
-msgid "Ik maak een oefenstudie aan"
+msgid "Ik maak een oefenaanvraag aan"
 msgstr "I am creating a practice proposal"
 
 #: proposals/forms.py:246 proposals/models.py:386
-msgid "Te kopiëren studie"
+msgid "Te kopiëren aanvraag"
 msgstr "Application to be copied"
 
 #: proposals/forms.py:248 proposals/models.py:388
-msgid "Dit veld toont enkel studies waar je zelf een medeuitvoerende bent."
+msgid "Dit veld toont enkel aanvragen waar je zelf een medeuitvoerende bent."
 msgstr "This field shows only applications in which you are involved."
 
 #: proposals/forms.py:288 proposals/forms.py:323
-msgid "Je kan de titel van je studie nu, indien nodig, wijzigen."
-msgstr "You can, if necessary, change the title of your study here."
+msgid "Je kan de titel van je aanvraag nu, indien nodig, wijzigen."
+msgstr "You can, if necessary, change the title of your proposal here."
```

Comment:
Generally speaking, I chose proposal when facing a submitter and referring to the _research proposal_. Application is the default, and always the case when referring to the application or application process. But I'm starting to agree with the consistency argument now.

"Study" is an absolute last resort when the rest just doesn't make sense because we're speaking about part of the hypothetical study as it may take place in the future.